### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by aarontmo** [jcreinhold/intensity-normalization#75] [Q&A] [answered]
Hello,

As I understand it, this library only works with python 3.9. Will there be an update so that it works with newer versions of python?

Thanks

**Comment by jcreinhold** [comment, marked as answer]
This package currently only works for Python v3.9. Install a conda or some other virtual environment with that version of Python and try installing again. This will also only work with Mac/Linux; it doesn't support Windows (or at least hasn't been tested on Windows). Create a virtual machine with a Linux distribution if you're using Windows.

If you're using Mac/Linux and using Python 3.9 and still encountering this error, then file a bug report issue (filling out the details in the form).

Let me know if the above info solves your problem.
